### DISCUSSION TITLE
Pattern organizer for a computer (Mac)

### DISCUSSION THREAD

**Discussion by tripsylon** [bangcorrupt/hacktribe#296] [Q&A] [answered]
Hello is there a way to organize your patterns on the computer I have a ton of patterns in a few folders and want to organize a live set structure and doing that within the electribe is a bit tricky i think. Thank you

**Comment by bangcorrupt** [comment, marked as answer]
@konsumer was working on this in [electribe_tools](https://github.com/konsumer/electribe_tools).  See https://github.com/bangcorrupt/hacktribe-editor-legacy/discussions/8 and try it [here](https://electribe_tools.surge.sh/).

**Reply by tripsylon** [reply]
This looks great I’m currently out of time but when i get the chance I’m really looking forward to try these out . Thank You 🙏.
